app/aggregates/reviewing/available_reviewer_actions.rb

```diff
@@ -23,14 +36,17 @@ def initialize(state:, application_type:, work_stream:, has_decisions:)
     attr_reader :state, :application_type, :work_stream, :has_decisions
 
     def actions
-      result = AVAILABLE_ACTIONS.dig(review_type, state).dup || []
-      result.prepend(:submit_decision) if funding_decision_actions_enabled?
-
-      result
+      config.dig(review_type, state).dup || []
```

Comment:
I think the `.dup` is no longer needed here. I only added it because I was modifying the hash.